cypress/e2e/signIn.cy.js

```diff
@@ -10,29 +10,31 @@ const homePage = new HomePageObject();
 describe('Sign In page', () => {
   let user;
 
-  before(() => {
+  const wrongEmail = '123' + user.email;
+  const loginFailedMessage = "Login failed";
+
+  beforeEach(() => {
     cy.task('db:clear');
     cy.task('generateUser').then(generateUser => {
       user = generateUser;
     });
+    cy.register(user.email, user.username, user.password);
   });
 
   it('should provide an ability to log in with existing credentials', () => {
     signInPage.visit();
-    cy.register(user.email, user.username, user.password);
 
-    signInPage.emailField
-      .type(user.email);
-    signInPage.passwordField
-      .type(user.password);
-    signInPage.signInBtn
-      .click();
+    signInPage.fillEmailField(user.email);
+    signInPage.fillPasswordField(user.password);
+    signInPage.clickOnSignInBtn;
```

Comment:
The 'signInBtn' is not being clicked, but simply referenced. Ensure a function call is made to perform the click action.